.github/workflows/backend.yml

```diff
@@ -1,70 +1,19 @@
 name: Backend
 
 on:
-  push:
-    branches:
-      - "master"
-      - "release-**"
-  pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+  workflow_call:
+    inputs:
+      skip:
+        type: boolean
+        default: false
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref && github.ref || github.run_id }}
+  group: ${{ github.workflow }}-${{ github.head_ref && github.ref || github.run_id }}-backend
```

Comment:
Isn't `github.workflow` already giving us the workflow name?

Reply:
it only gives the top level workflow name - so this will be `run-tests` - when i first ran this all the sub-workflows cancelled each other 🫠

Reply:
Interesting. I didn't know it goes all the way to the top.
Thanks for sharing.